### PR TITLE
Replace logger with bunyan - Closes #2148

### LINKED ISSUE #2148: Improve logging system
### Description
Currently, logging does not have standard and outputted in text format.
It is hard to use for analysis of KPI.

### Expected
- console log in the text format for readability
- file log in the JSON format for extensible usage
- Standardised format and keys for better searchability 

### Which version(s) does this affect? (Environment, OS, etc...)
1.0

### PR DESCRIPTION
**Pull request by shuse2**
### What was the problem?
Logging was in plain text format, where it was hard to use for analysis

### How did I fix it?
Change logger to JSON format

### How to test it?
start application

### Review checklist

* The PR resolves #2148
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated

### CODE REVIEW

File: framework/src/components/storage/adapters/pgp_adapter.js [hunk 1]
```diff
@@ -88,7 +88,7 @@ class PgpAdapter extends BaseAdapter {
 
 		monitor.attach(this.pgpOptions, this.options.logEvents);
 		monitor.setLog((msg, info) => {
-			this.logger.log(info.event, info.text);
+			this.logger.debug(info.event, info.text);
```
Comment (limiaspasdaniel) [inline framework/src/components/storage/adapters/pgp_adapter.js]: Shouldn't this be `info`? As in framework/src/controller/application.js#341

File: lisk/src/index.js [hunk 2]
```diff
@@ -118,6 +118,6 @@ try {
 			process.exit();
 		});
 } catch (e) {
-	console.error('Application start error.', e.errors);
+	console.error('Application start error.', e);
```
Comment (limiaspasdaniel) [inline lisk/src/index.js]: Why this change? Just curious

Comment (shuse2) [reply lisk/src/index.js]: Ah this shouldnt be commited (not related to this PR)
but it shows undefined, because this error doesn't have `errors` property.
It will be fixed in other PR

File: framework/src/components/storage/adapters/pgp_adapter.js (hunk 1 above)
Comment (shuse2) [reply framework/src/components/storage/adapters/pgp_adapter.js]: I think info level is more for `user` so DB related stuff should be almost always `debug`

File: framework/test/test_app/index.js [hunk 3]
```diff
@@ -36,7 +36,7 @@ try {
 
 	app
 		.run()
-		.then(() => app.logger.log('App started...'))
+		.then(() => app.logger.trace('App started...'))
```
Comment (nazarhussain) [inline framework/test/test_app/index.js]:
```suggestion
		.then(() => app.logger.info('App started...'))
```

**Comment by nazarhussain** [comment]
@shuse2  To see the logs properly in the console in human readable format, Bunyan have a utility for it:

https://github.com/trentm/node-bunyan#cli-usage

```
$ node hi.js | bunyan -l warn
[2013-01-04T19:08:37.182Z]  WARN: myapp/40353 on banana.local: au revoir (lang=fr)
```

I suggest we should provide a guide fro developers of the framework as well users of SDK with a bit of that so they can read the logs easily.
